.github/workflows/execute_advanced_tests.yaml

```diff
@@ -63,14 +67,14 @@ jobs:
 
   e2e-upgrade-test:
     if: ${{ github.event.inputs.e2e-upgrade-test == 'true' || github.event_name == 'schedule' }}
-    runs-on: buildjet-4vcpu-ubuntu-2204
+    runs-on: buildjet-16vcpu-ubuntu-2204
     timeout-minutes: 120
     steps:
       - name: "Checkout Code"
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
 
       - name: Start Test
-        run: make start-upgrade-test
+        run: make start-e2e-import-mainnet-upgrade
```

Comment:
Please add a new job step rather than modifying this one, we need this one to be fast.

Reply:
@gzukel  can you resolve this